php/modules/Session.php

```diff
@@ -99,11 +112,18 @@ public function login($username, $password) {
 		$username = crypt($username, $this->settings['username']);
 		$password = crypt($password, $this->settings['password']);
 
-		# Check login with crypted hash
 		if ($this->settings['username']===$username&&
 			$this->settings['password']===$password) {
 				$_SESSION['login']		= true;
 				$_SESSION['identifier']	= $this->settings['identifier'];
+
+				$expire = time() + 60 * $this->settings['sessionLength'];
```

Comment:
I added a default value of 24 hours to the settings_content.sql. Though I'm not sure about the update script since I don't know which version I should call it. It is done, and can be found in myu master branch, if a merge where to happen :)